--- origsrc/ilmbase-1.0.3/IlmThread/IlmThreadMutexPosix.cpp	2012-07-17 18:27:25.000000000 -0500
+++ src/ilmbase-1.0.3/IlmThread/IlmThreadMutexPosix.cpp	2013-11-26 16:52:05.094806900 -0600
@@ -41,7 +41,7 @@
 
 #include "IlmBaseConfig.h"
 
-#if HAVE_PTHREAD
+#if HAVE_PTHREAD && !defined(_WIN32) && !defined(_WIN64)
 
 #include "IlmThreadMutex.h"
 #include "Iex.h"
--- origsrc/ilmbase-1.0.3/IlmThread/IlmThreadPosix.cpp	2012-07-17 18:27:25.000000000 -0500
+++ src/ilmbase-1.0.3/IlmThread/IlmThreadPosix.cpp	2013-11-26 16:52:18.919597700 -0600
@@ -41,7 +41,7 @@
 
 #include "IlmBaseConfig.h"
 
-#if HAVE_PTHREAD
+#if HAVE_PTHREAD && !defined(_WIN32) && !defined(_WIN64)
 
 #include "IlmThread.h"
 #include "Iex.h"
--- origsrc/ilmbase-1.0.3/IlmThread/IlmThreadSemaphorePosix.cpp	2012-07-17 18:27:25.000000000 -0500
+++ src/ilmbase-1.0.3/IlmThread/IlmThreadSemaphorePosix.cpp	2013-11-26 16:52:51.973488200 -0600
@@ -41,7 +41,7 @@
 
 #include "IlmBaseConfig.h"
 
-#if HAVE_PTHREAD && HAVE_POSIX_SEMAPHORES
+#if HAVE_PTHREAD && HAVE_POSIX_SEMAPHORES && !defined(_WIN32) && !defined(_WIN64)
 
 #include "IlmThreadSemaphore.h"
 #include "Iex.h"
--- origsrc/ilmbase-1.0.3/IlmThread/IlmThreadSemaphorePosixCompat.cpp	2012-07-17 18:27:25.000000000 -0500
+++ src/ilmbase-1.0.3/IlmThread/IlmThreadSemaphorePosixCompat.cpp	2013-11-26 16:52:59.417914000 -0600
@@ -42,7 +42,7 @@
 
 #include "IlmBaseConfig.h"
 
-#if HAVE_PTHREAD && !HAVE_POSIX_SEMAPHORES
+#if HAVE_PTHREAD && !HAVE_POSIX_SEMAPHORES && !defined(_WIN32) && !defined(_WIN64)
 
 #include "IlmThreadSemaphore.h"
 #include "Iex.h"
--- origsrc/ilmbase-1.0.3/IlmThread/Makefile.am	2012-07-17 18:27:25.000000000 -0500
+++ src/ilmbase-1.0.3/IlmThread/Makefile.am	2013-11-26 16:54:46.307027800 -0600
@@ -8,7 +8,8 @@ libIlmThread_la_SOURCES = IlmThreadPool.
 			  IlmThreadSemaphore.cpp IlmThreadMutex.cpp \
 			  IlmThreadPosix.cpp IlmThreadSemaphorePosix.cpp \
 			  IlmThreadSemaphorePosixCompat.cpp \
-			  IlmThreadMutexPosix.cpp
+			  IlmThreadSemaphoreWin32.cpp IlmThreadWin32.cpp \
+			  IlmThreadMutexPosix.cpp IlmThreadMutexWin32.cpp
 
 libIlmThread_la_LDFLAGS = -version-info @LIBTOOL_VERSION@ -no-undefined
 libIlmThread_la_LIBADD = ../Iex/libIex.la
@@ -20,7 +21,4 @@ libIlmThreadinclude_HEADERS = IlmThreadP
 
 noinst_HEADERS = 
 
-EXTRA_DIST = $(noinst_HEADERS) IlmThreadMutexWin32.cpp IlmThreadSemaphoreWin32.cpp \
-	     IlmThreadWin32.cpp
-
-INCLUDES = -I$(top_builddir) -I$(top_srcdir)/Iex -I$(top_srcdir)/config
\ No newline at end of file
+INCLUDES = -I$(top_builddir) -I$(top_srcdir)/Iex -I$(top_srcdir)/config
